scripts/dropconfig

```diff
@@ -0,0 +1,306 @@
+#!/usr/bin/env python
+
+#####################################################################
+#
+# dropconfig is a tool for configuring drop counters.
+#
+#####################################################################
+
+# FUTURE IMPROVEMENTS
+# - Add more filters to the show commands (e.g. filter by alias, filter by name, etc.)
+# - Add the ability to change user-only attributes like alias, group, description
+
+import swsssdk
+import argparse
+import os
+
+from tabulate import tabulate
+
+# CONFIG_DB Tables
+DEBUG_COUNTER_CONFIG_TABLE = 'DEBUG_COUNTER'
+DROP_REASON_TABLE_PREFIX = 'DEBUG_COUNTER_DROP_REASON'
+
+# STATE_DB Tables
+DEBUG_COUNTER_CAPABILITY_TABLE = 'DEBUG_COUNTER_CAPABILITIES'
+
+# Drop Counter Configuration Headers
+drop_counter_config_header       = ['Counter', 'Alias', 'Group', 'Type', 'Reasons', 'Description']
+drop_counter_capability_header = ['Counter Type', 'Total']
+
+class DropConfig(object):
+    def __init__(self):
+        self.config_db = swsssdk.ConfigDBConnector()
+        self.config_db.connect()
+
+        self.state_db = swsssdk.SonicV2Connector(host='127.0.0.1')
+        self.state_db.connect(self.state_db.STATE_DB)
+
+    # -c show_config 
+    def print_counter_config(self, group):
+        """
+            Prints out the configuration for all counters that are currently set up
+        """
+
+        table = []
+        for counter in self.get_config(group):
+            table.append((counter.get('name', ''), 
+                          counter.get('alias', ''),
+                          counter.get('group', ''),
+                          counter.get('type', ''),
+                          counter.get('reason', ''),
+                          counter.get('description', '')))
+        print(tabulate(table, drop_counter_config_header, tablefmt='simple', stralign='left'))
+
+    def print_device_capabilities(self):
+        """
+            Prints out the capabilities that this device has
+        """
+
+        device_caps = self.get_device_capabilities()
+
+        table = []
+        for counter, capabilities in device_caps.iteritems():
+            table.append((counter, capabilities.get('count', 'N/A')))
+        print(tabulate(table, drop_counter_capability_header, tablefmt='simple', stralign='left'))
+
+        for counter, capabilities in device_caps.iteritems():
+            supported_reasons = deserialize_reason_list(capabilities.get('reasons', ''))
+            if supported_reasons:
+                print('\n{}'.format(counter))
+                for reason in supported_reasons:
+                    print('\t{}'.format(reason))
+    
+    def create_counter(self, counter_name, alias, group, counter_type, description, reasons):
```

Comment:
How about user wanted to update any reasons to the existing countes? nice to have some method to do that?